voxblox/src/integrator/tsdf_integrator.cc

```diff
@@ -372,9 +386,10 @@ void MergedTsdfIntegrator::bundleRays(
 void MergedTsdfIntegrator::integrateVoxel(
     const Transformation& T_G_C, const Pointcloud& points_C,
     const Colors& colors, bool enable_anti_grazing, bool clearing_ray,
-    const std::pair<GlobalIndex, AlignedVector<size_t>>& kv,
+    const GlobalIndex& voxel_index,
+    const AlignedVector<size_t>& point_C_indices,
```

Comment:
I'm not sure why is this aligned, but ok...

Reply:
yes, me neither, but it's used in other parts of the code as well, so probably it's better to leave it as it is for now.